### PR TITLE
Use `verlib2.Version` for comparing versions

### PR DESCRIPTION
**Pull request by amotl**
## About

For not duplicating corresponding code any longer, there is the [verlib2 package](https://pypi.org/project/verlib2/) now, which includes the same canonical code from `packaging.version` as before, see GH-513.

### CODE REVIEW

**Review by seut — DISMISSED** [review]
👍

**Review by seut — COMMENTED** [review]
great! but it seems not to work as expected as a lot of test fail?

File: src/crate/client/_pep440.py [hunk 1]
```diff
@@ -1,501 +0,0 @@
...
-    @property
-    def is_postrelease(self):
-        return bool(self._version.post)
-
-    @property
-    def version(self) -> tuple:
-        """
-        PATCH: Return version tuple for backward-compatibility.
-        """
-        return self._version.release
```
Comment (amotl) [inline src/crate/client/_pep440.py:403]: @seut: This little polyfill needs to be added to verlib2, to make it compatible.

Comment (amotl) [reply src/crate/client/_pep440.py:403]: The snippet to gain backwards-compatibility has been added with verlib2 0.2.0. It resolves the issue of failing test cases.

**Review by seut — APPROVED** [review]

**Pull request merged**